.github/workflows/auto-reply.yml

```diff
@@ -0,0 +1,51 @@
+name: Issue Reply
+on:
+  issues:
+    types: [labeled]
+jobs:
+  reply-helper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: help wanted
+        if: github.event.label.name == 'help wanted'
+        uses: actions-cool/issues-helper@v1.2
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. welcome to commit your issue here , we will check and reply you as soon.
+            你好 @${{ github.event.issue.user.login }}，非常欢迎提交你的问题和需求，我们会尽快确认并回复你~~
+
+      - name: bug
+        if: github.event.label.name == 'bug'
+        uses: actions-cool/issues-helper@v1.2
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. welcome to commit your issue here , we will check and reply you as soon.
+            你好 @${{ github.event.issue.user.login }}，非常欢迎提交你的问题和需求，我们会尽快确认并回复你~~
+
+      - name: question
+        if: github.event.label.name == 'question'
+        uses: actions-cool/issues-helper@v1.2
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. welcome to commit your issue here , we will check and reply you as soon.
```

Comment:
Hello @${{ github.event.issue.user.login }}. Welcome to commit your issue here, we will check and reply to you as soon as possible.